resources/views/forum/topics/_post.blade.php

```diff
@@ -19,6 +19,13 @@
     }
 
     $user = $post->userNormalized();
+    $group = isset($topic) && $user->isModeratorForForum($topic->forum)
+        ? (object)[
+            'colour' => 'hsl(200, 60%, 50%)',
```

Comment:
Where's this colour coming from?

Reply:
https://github.com/ppy/osu-web/blob/b460e4853e0689420b7888a33d106b8ee33c1eff/resources/assets/lib/beatmap-discussions/mapper-group.ts#L7
was just to get it working, idk if that makes sense for design